packages/core/lib/console.js

```diff
@@ -187,7 +207,7 @@ class Console extends EventEmitter {
     };
   }
 
-  provision() {
+  provision(initialProvision = false) {
```

Comment:
Would this cover the case when develop is started and there are no artifacts in the build folder? The build folder can be populated on a `migrate` or `compile` after this first check.

Reply:
Just verified this is still a clobber path

## Reprod steps

### Create project and start develop REPL
```console
truffle init
truffle create all Buffer
truffle develop
```

## REPL commands to clobber Buffer
```console
truffle(develop)> Buffer
[Function: Buffer] {
  poolSize: 8192,
  from: [Function: from],
  of: [Function: of],
  alloc: [Function: alloc],
  allocUnsafe: [Function: allocUnsafe],
  allocUnsafeSlow: [Function: allocUnsafeSlow],
  isBuffer: [Function: isBuffer],
  compare: [Function: compare],
  isEncoding: [Function: isEncoding],
  concat: [Function: concat],
  byteLength: [Function: byteLength],
  [Symbol(kIsEncodingSymbol)]: [Function: isEncoding]
}
truffle(develop)> migrate
## deployment steps...

truffle(develop)> Buffer.abi
[ { inputs: [], stateMutability: 'nonpayable', type: 'constructor' } ]
truffle(develop)>


```